script/install/conda_eva_environment.yml

```diff
@@ -27,10 +27,11 @@ dependencies:
   - mock
   - pytest-asyncio
   - flake8
-  - sphinx 
+  - sphinx
   - sphinx_rtd_theme
   - pip:
     - antlr4-python3-runtime==4.8
+    - pyspark==3.0.2
     - petastorm
```

Comment:
afaik, pyspark is a dependency of petastorm. So let's also freeze the version of petastorm.

Reply:
Sure. Should we keep it as a separate PR or just do with inside Rodrigo's PR?

Reply:
Actually, done